src/components/orders/AmountsDisplay/index.tsx

```diff
@@ -38,9 +40,7 @@ function Row(props: RowProps): JSX.Element {
       <RowContents>
         <span>{formattedAmount}</span>
         {/* <UsdAmount>(~${usdAmount})</UsdAmount> */}
```

Comment:
bye bye :wave:

Reply:
unless placeholder for when we do this, then nvm

Reply:
No no no, this stays!